src/action-bar.ts

```diff
@@ -238,6 +238,13 @@ export class AdaptiveElement extends Base {
   public get hasItems(): boolean {
     return (this.items || []).length > 0;
   }
+  public getItemById(id: string): IActionBarItem {
+    if (!this.items) return null;
+    for (var i = 0; i < this.items.length; i++) {
+      if (this.items[i].id === id) return this.items[i];
+    }
+    return null;
+  }
 
```

Comment:
The src/action-bar.ts file contains deprecated code. We'll remove all the classes from this file. We need to use classes from the src/actions folder. I forgot to tell you.

Reply:
Arh, OK. I have removed it. I wanted to use this function in Creator V2. Nevermind, I will add it later.

Thank you,
Andrew